app/server.py

```diff
@@ -277,6 +277,36 @@ def restock_action(prod_id):
     Restock new quantity.
 
     This endpoint will update the number of new_qty of the given prod_id.
+    ---
+    tags:
+        -   Inventory
+    consumes:
+        -   application/json
+    definitions:
+        Restock_Amount:
+            type: object
+            properties:
+                restock_amt:
+                    type: integer
+                    minimum: 0
```

Comment:
I guess for restocking, the minimum amount should be bigger than 0?

Reply:
Our current restock function supports any restock_amt >= 0.

The restock_amt in the screenshot is manually typed in. There is no default value for that. The response should be 400 when restock_amt is -1. It's just a demonstration.